react/providers/CozyTheme/index.jsx

```diff
@@ -25,7 +28,30 @@ export const useCozyTheme = () => {
   return context
 }
 
-const CozyTheme = ({ variant, className, ignoreItself, children }) => {
+const CozyThemeQuery = props => {
+  const instanceQuery = buildSettingsInstanceQuery()
+  const { data: instance, ...instanceQueryLeft } = useQuery(
+    instanceQuery.definition,
+    instanceQuery.options
+  )
+
+  if (
+    isQueryLoading(instanceQueryLeft) &&
+    !hasQueryBeenLoaded(instanceQueryLeft)
```

Comment:
It is useful to check if query is loading AND if not query been loaded ?

Reply:
c'est pour éviter de repasser à null après expiration de la fetchpolicy avec un fetch en arrière plan